### PR TITLE
ENG-733: Update workers AMI for PXB.cd

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Deployed on pxb

### CODE REVIEW

**Review by delgod — APPROVED** [review]

File: IaC/pxb.cd/init.groovy.d/cloud.groovy [hunk 1]
```diff
@@ -215,7 +213,7 @@ initMap['min-buster-x64'] = '''
 initMap['min-bionic-x64'] = '''
     set -o xtrace
     if ! mountpoint -q /mnt; then
-        DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme0n1 | head -1)
+        DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme1n1 | head -1)
```
Comment (EvgeniyPatlan) [inline IaC/pxb.cd/init.groovy.d/cloud.groovy:216]: Why this change appeared?

Comment (Sudokamikaze) [reply IaC/pxb.cd/init.groovy.d/cloud.groovy:216]: Because I've changed AMI ID to newer one and I got an error that nvme0n1 being used

Comment (delgod) [reply IaC/pxb.cd/init.groovy.d/cloud.groovy:216]: can we make it like this 
```
DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme0n1 /dev/nvme1n1 | head -1)
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: IaC/pxb.cd/init.groovy.d/cloud.groovy (hunk 1 above)
Comment (EvgeniyPatlan) [reply IaC/pxb.cd/init.groovy.d/cloud.groovy:216]: yeah the suggested approach should be better as we always face this issue and change device name back and forth

Comment (Sudokamikaze) [reply IaC/pxb.cd/init.groovy.d/cloud.groovy:216]: It won't work, as it'll pick the first available device, then will try to create an ext2 FS
and will stuck on creation

**Comment by Sudokamikaze** [comment]
Please re-check, PR was rebased

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by delgod — APPROVED** [review]
